### PR TITLE
Changes Made

### PR DESCRIPTION
**Pull request by puranchandralohar**
* Updated Population Chart With Apexs Chart
* Added Richest Person Table
* Added a World Map
* Currency Converter

### CODE REVIEW

File: src/components/ExchangeRate/Converter.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  const { currencies, base, amount, convertTo, result, date } = initialState;
+
+  useEffect(() => {
+    if (amount === isNaN) {
+      return;
+    } else {
+      axios
+        .get(`https://open.er-api.com/v6/latest/${base}`)
+        .then((responce) => {
+          console.log(responce.data);
```
Comment (Souvikdeb2612) [inline src/components/ExchangeRate/Converter.jsx]: remove/comment console statements before pushing

File: src/components/RichestPerson/RichestPerson.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              <th>Person</th>
+              <th>Rank</th>
+              <th>Name</th>
+              <th>Source</th>
+              <th>Net Worth</th>
+              <th>Country</th>
+            </tr>
+          </thead>
+          <tbody>
+            {data.map((value) => {
```
Comment (Souvikdeb2612) [inline src/components/RichestPerson/RichestPerson.jsx]: directly destructure here

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/ExchangeRate/Converter.jsx (hunk 1 above)
Comment (puranchandralohar) [reply src/components/ExchangeRate/Converter.jsx]: removed .

File: src/components/RichestPerson/RichestPerson.jsx (hunk 2 above)
Comment (puranchandralohar) [reply src/components/RichestPerson/RichestPerson.jsx]: done

File: src/components/Population/Population.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    useEffect(()=>{
+      const country=[];
+      const population =[];
+      const url = `https://raw.githubusercontent.com/puranchandralohar/apis/main/top10country.json`
+      axios
+      .get(url)
+      .then((responce)=>{
+          let result = responce.data;
+          result.map((item)=>{
```
Comment (Souvikdeb2612) [inline src/components/Population/Population.jsx:31]: item should be destructured here

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**